README.rst

```diff
@@ -49,12 +49,19 @@ The -L flag can be used to whitelist certain words that are comma-separated plac
 
     codespell -S, --skip=
 
-Comma-separated list of files to skip. It accepts globs as well.  Examples:
+The -S (--skip) flag is used to skip over comma-separated list of files and/or directories. It accepts globs as well. **Important note:** do not append a '/' to a directory you want to skip. This currently renders the entry ineffective.
```

Comment:
Ooops! I forgot the article.
"skip over comma-separated"
should read
"skip a over comma-separated"

Reply:
huh? `skip a over comma-separated` that makes no sense to me :confused:

Reply:
"skip over a comma-separated"